thirdparty/runtime_environments.go

```diff
@@ -96,3 +96,56 @@ func (c *RuntimeEnvironmentsClient) getOSInfo(ctx context.Context, amiID string,
 	}
 	return osInfo, nil
 }
+
+// Toolchain represents a toolchain's information.
+type Toolchain struct {
+	Name    string
+	Version string
+	Manager string
+}
+
+// ToolchainFilterOptions represents the filtering arguments, each of which is optional except for the AMIID.
+type ToolchainFilterOptions struct {
+	AMIID   string
```

Comment:
maybe we should consider calling this just `AMI` since `AMIID` is hard to read (and same for the `PackageFilterOptions`)